src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1085,6 +1095,27 @@ public WrapModeType WrapModeV
             }
         }
 
+        /// <summary>
+        /// Gets or sets the mode to adjust view size to preserve the aspect ratio of the image resource.
+        /// If this is set to be true, then the width or height, which is not set by user explicitly, can be adjusted to preserve the aspect ratio of the image resource.
+        /// AdjustViewSize works only if ImageView is added to a View having Layout.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AdjustViewSize
```

Comment:
We usually puts IsXXX in boolean data, but I'm not sure IsAdjustedViewSize is correct or not...........

Reply:
I understand your point.
I think that "Adjust" is suitable for this case.
Based on the above, we can use IsAdjustingViewSize, but IMHO, AdjustViewSize looks better.